### ISSUE TITLE
1.2.3 and 1.2.5 (Audio Description) proposed Note based on EN 301 549 deliberations

### ISSUE THREAD

**Issue by daniel-montalvo** [w3c/wcag2ict#786]
At [This ETSI issue](https://labs.etsi.org/rep/HF/en301549/-/issues/595#note_26974) they are proposing to add the following note to EN 301 549. They are asking us to include it to WCAG2ICT as well.

> NOTE: Audio descriptions add descriptive information of important visual information, to the extent possible in the gaps of the audio track of the video. This is usually implemented by providing a second audio track for the audio-video media. However, this requirement could also be met by fully describing important visual information in the main audio track. Doing so eliminates the work of adding a second audio track and finding appropriate gaps in the audio to add descriptions of visual information. This requirement would also be met if there is no important visual information to be described in the audio (that is not already in the audio)., or if there are few or even zero gaps available during the video and during the credits (if any) before the video (since all gaps were utilized).

W3C/WAI would not be comfortable adding this note as-is because it is making prevail the assumption that having a video without gaps and needing audio description would automatically pass, which is not stated in WCAG. This would be effectively re-interpreting one WCAG requirement, which is also out of scope for WCAG2ICT.

So WCAG2ICT initially agreed in its [Extra Friday meeting](https://www.w3.org/2025/09/26-wcag2ict-minutes.html) to remove the excerpt "or if there are few or even zero gaps available during the video and during the credits (if any) before the video (since all gaps were utilized)".

This is to record my proposed additional edits to the Note. I am told this is important for EN to improve understanding of the requirement, but I think the current state of the note doesn't really help to achieve this goal and would introduce confusion if added to WCAG2ICT as-is.

I propose that the note read:

> NOTE: Audio descriptions (also called "video descriptions" and "described videos") add descriptive information of important visual information needed to understand the video content, including text displayed in the video. This requirement can be met by fully describing important visual information in the main audio track of the video, which eliminates the need for audio descriptions altogether. When audio descriptions are needed, one way to implement them is by providing a second audio track for the audio-video media. 

Edits include:

- Added other common terms for audio description.
- Removed "to the extent possible". Most of this stuff is by design "to the extent possible". I don't think that's crutial to the note.
- Tweaked the definition according to what the audio/video media guide currently has, which has been vetted most recently than the understanding docs.
- Flipped around the next two sentences: best practice is to integrate descriptions in the main audio track, if not, then continue with what we had.
- Added "of the video" to "main audio track" to avoid ambiguity.
- Removed remaining references to the gaps.
- Changed eliminates the work of adding a second audio track and finding appropriate gaps in the audio to add descriptions of visual information" to the upfront more clearer "eliminates the need for audio descriptions altogether".
- Changed "are usually implemented" to "one way to implement them"
- Removed "This requirement would also be met if there is no important visual information to be described in the audio (that is not already in the audio)". This is repeated and can be expressed much clearer in the best practice sentence "eliminates the need of audio descriptions altogether".

**Comment by daniel-montalvo** [comment]
Hi @GreggVan 

Thanks for your reply to this thread.

I was able to stay in the WCAG2ICT meeting until the end, so I am not sure why this copy you just put above is slightly different than the one we had on the [WCAG2ICT working google doc](https://docs.google.com/document/d/1VrVZkHNcoJ-DOswwCf98HBZrOqLWHhzk51kJqdqM93Y/edit?tab=t.0#heading=h.iajq99hc9s85). In any case, thanks for putting it here, much appreciated!

In any case, my proposed edits still stand based on this other proposal that you mention above Mainly:

* We should acknowledge the use of other terms for audio description, such as "video description" or "described videos"
* There is a missing "including text displayed in the video" to the current definition of audio description that is relevant
* I still don't think we should mention gaps at all. The audio description could be consumed via a synched timed-stamped file that the user is reading with their braille display, gaps would not matter there. As stated previously, gaps are indeed only one way in which you could serve audio description. In a high-level note, which purpose is improving understanding of the requirement, this would add confusion instead of clarifying
* Also in terms of clarity, it would greatly help if we flipped around the intermediate sentences. First tell me how I can meet the requirement (best practice of fully describing the info in the main audio track of the video) and then get into the details of how I should implement audio description.
* Also, in terms of implementing audio description, I doubt that the main audiences of these two documents will be implementing audio descriptions themselves. Rather, that's something they'll most likely outsource.

**Comment by GreggVan** [comment]
> On Sep 28, 2025, at 6:05 AM, Daniel Montalvo ***@***.***> wrote:
> 
> 
> daniel-montalvo
>  left a comment 
> (w3c/wcag2ict#786)
>  <https://github.com/w3c/wcag2ict/issues/786#issuecomment-3343285412>
> Hi @GreggVan <https://github.com/GreggVan>
> Thanks for your reply to this thread.
> 
> I was able to stay in the WCAG2ICT meeting until the end, so I am not sure why this copy you just put above is slightly different than the one we had on the WCAG2ICT working google doc <https://docs.google.com/document/d/1VrVZkHNcoJ-DOswwCf98HBZrOqLWHhzk51kJqdqM93Y/edit?tab=t.0#heading=h.iajq99hc9s85>. In any case, thanks for putting it here, much appreciated!
> 
> In any case, my proposed edits still stand based on this other proposal that you mention above Mainly:
> 
> We should acknowledge the use of other terms for audio description, such as "video description" or "described videos"
    GV: agree.  but that is usually done in a definition - not a note.  for example in EN 301 549

audio description <>: narration added to the soundtrack to describe important visual details of synchronised audio-visual content, which cannot be understood from the main soundtrack alone

NOTE 1:  This is also variously described using terms such as "video description" or variants such as "descriptive narration".

NOTE 2:  Audio description helps blind persons, persons with low vision, and persons who need assistance with understanding important facial expressions and other visual content to perceive and understand the visual content of audio-visual media.

NOTE 3:  Audio description can provide information about actions, characters, scene changes, on-screen text, and other visual content.

NOTE 4:  The audio description is inserted in the gaps in the main audio narrative. It typically uses a different voice, tone, or speed to separate it from the main audio.   
> There is a missing "including text displayed in the video" to the current definition of audio description that is relevant
    GV: Hmmm.   if it is important text - it would be important visual information and already covered (and done in practice)   but there are tons of incidental text in video.  So I  think this would be over prescriptive.  
> I still don't think we should mention gaps at all. The audio description could be consumed via a synched timed-stamped file that the user is reading with their braille display, gaps would not matter there. As stated previously, gaps are indeed only one way in which you could serve audio description. In a high-level note, which purpose is improving understanding of the requirement, this would add confusion instead of clarifying
   GV: the audio is audio not text.  There is no way for a braille display to display an audio signal.  So I  do not know what you mean.    Also that simply is not the definition of audio description.     
> Also in terms of clarity, it would greatly help if we flipped around the intermediate sentences. First tell me how I can meet the requirement (best practice of fully describing the info in the main audio track of the video) and then get into the details of how I should implement audio description.
    GV: could be.   I think in this case it is easier as is but that is just me.  Happy to go with what the group decides.  However this is all going to become cement in 2 days.    If this is critical - push it to Mary Jo.   If just “somewhat better to read” I  think we are too late.    
> Also, in terms of implementing audio description, I doubt that the main audiences of these two documents will be implementing audio descriptions themselves. Rather, that's something they'll most likely outsource.
    GV: whether they do it or specify it for others - that is up to them.   But it is they are are responsible that serve/sell it.    So wouldn't they be the primary audience since the people the contract with are secondary as they are not required to do this — they are just doing it for the primary. So wouldn't the thing they pay attention to be the language of the work order sent to them?

**Comment by maryjom** [comment]
There's also a WCAG technique [G8: Providing a movie with extended audio descriptions](https://www.w3.org/WAI/WCAG22/Techniques/general/G8) for extended audio description where a separate audio track is provided - another way to meet this requirement so maybe text for the first sentence should be left ambiguous as well by removing: ", to the extent possible in the gaps of the audio track of the video"

**Comment by GreggVan** [comment]
Yes.

Sorry,   I  meant to remove that comma.  It is essential  or this just becomes  “do this if it is possible”   which makes it a recommendation  - not requirement

G

**Comment by daniel-montalvo** [comment]
That's the edit I think is necessary for us at W3C/WAI to be able to "live with" this note. The others are more stylistic, we could live without. But these references to the gaps should be reemoved.

@GreggVan @mapluke I'd like to extend these comments to the EN GitLab repository but I am no longer able to access my account and the ETSI support is not getting back to me with steps to recover my acccess.

**Comment by daniel-montalvo** [comment]
> I posted this to GitLab
> 
> > NOTE  Daniel Montalvo lost access to GitLab -- and wanted to post this to this list
> > > That's the edit [removal of the comma before 'as is possible'] I think is necessary for us at W3C/WAI to be able to "live with" this note. The others are more stylistic, we could live without. But these references to the gaps should be removed.

Thanks for your help, but that was not the edit I was proposing @GreggVan 

The edit I was proposing is to remove ", to the extent possible in the gaps of the audio track of the video"altogether, as @maryjom  suggested in [this comment above](https://github.com/w3c/wcag2ict/issues/786#issuecomment-3347592427).
.

> Daniel You aren't suggesting we require all videos to be extended audio descriptions - or to cut out dialog in the AV to include more audio description than will fit in the gaps are you?

I am not. I am just saying that the note still implies that the *only* way to provide audio descriptions is via the existing gaps in the main track of the video media. That is not accurate and can lead to confusion. Ultimately, it can lead to reinterpreting a WCAG Success Criteria under the EN 301 549, which I don't think is in scope for the EN as well.

**Comment by daniel-montalvo** [comment]
@GreggVan 

1. I don't see how a note (which is by definition **informative**) would extend a requirement.
2. I've seen many times (and I'm pretty sure you too based on your experience) how a note can be interpreted in a way that restricts a requirement. As I have said many times now, this one has the potential to be used in that way, and I oppose to letting this into the WCAG2ICT language.
3. Your point about the gaps, which I get is the most common way of doing audio descriptions, is already stated in the definitions of audio descriptions for EN and WCAG.

    EN -- "NOTE 4:	The audio description is inserted in the gaps in the main audio narrative. It typically uses a different voice, tone, or speed to separate it from the main audio".

    WCAG -- NOTE 2: "In standard audio description, narration is added during existing pauses in dialogue".

WCAG's definition qualifies "filling the gaps" to "standard audio description". The EN does not, which is also inaccurate.

Why should we add this to a note other than just to repeat ourselves and create confusion by inferring "filling the gaps" is the *only* method?

In a pre-recorded environment, if you have no gaps and description of visual information (audio description) is needed, you still can add it to the beginning/end of the existing video as another piece of pre-recorded audio (no need to pause the video).

You've been stating that any other action to include audio descriptions other than "filling the gaps" is "extended audio description". I also think that's inaccurate. Inserting the description at the beginning/end of the video won't be considered extended audio description according to the [extended audio descriptions WCAG definition](https://www.w3.org/TR/wcag/#dfn-extended-audio-description), because you wouldn't be pausing the video to accommodate for audio descriptions:

> audio description that is added to an audiovisual presentation by pausing the video so that there is time to add additional description
> NOTE
> This technique is only used when the sense of the video would be lost without the additional audio description and the pauses between dialogue/narration are too short.

**Comment by maryjom** [comment]
##Latest proposals
From what I can discern from the many threads of email between Daniel, Mike, Gregg and myself I think these are the two proposals on the table.
**Proposal 1** - combines Note 1, 2 and new note into a single note
>NOTE: Audio descriptions (also called "video descriptions" and "described videos") add descriptive information of important visual information needed to understand the video content, including text displayed in the video. This requirement can be met by fully describing important visual information in the main audio track of the video, which eliminates the need for audio descriptions altogether. When audio descriptions are needed, one way to implement them is by providing a second audio track for the audio-video media.

OR

**Proposal 2** - adds a new Note 3 (and removing note 2) - from the Friday 26 Sept. WCAG2ICT
>NOTE: Audio descriptions add descriptive information of important visual information, to the extent possible in the gaps of the audio track of the video. This is usually implemented by providing a second audio track for the audio-video media. However, this requirement could also be met by fully describing important visual information in the main audio track. Doing so eliminates the work of adding a second audio track and finding appropriate gaps in the audio to add descriptions of visual information. This requirement would also be met if there is no important visual information to be described in the audio (that is not already in the audio).

**Comment by GreggVan** [comment]
RE notes

You are correct -- a proper note cannot change a  provision.  But an improper one can appear to -- so standards do not allow improper notes.

Any draft note that extends the or alters rather than explains a requirement - therefore is not allowed. 

RE extended Audio Description 

V20 has now shipped.  It doesn't use my preferred language - and it won't address or stop the never-ending discussion about how much AD is enough when there are little or no gaps in dialog to put it in.  But we don't seem to have consensus on an answer to  that -- so we will be kicking the can down the road.   

Congrats everyone on working so hard to try go get this as good as we could.
